pkg/services/authn/authnimpl/service.go

```diff
@@ -155,7 +155,7 @@ func ProvideService(
 	s.RegisterPostAuthHook(userSyncService.SyncUserHook, 10)
 	s.RegisterPostAuthHook(userSyncService.EnableDisabledUserHook, 20)
 	s.RegisterPostAuthHook(orgUserSyncService.SyncOrgRolesHook, 30)
-	s.RegisterPostAuthHook(userSyncService.SyncLastSeenHook, 40)
+	s.RegisterPostAuthHook(userSyncService.SyncLastSeenHook, 120)
```

Comment:
```suggestion
	// Hook needs to run after FetchSyncedUserHook (priority 110) to have a populated identity struct
	s.RegisterPostAuthHook(userSyncService.SyncLastSeenHook, 120)
```

Reply:
with the second commit it actually doesn't need to run after `FetchSyncedUserHook`